Library/Homebrew/compilers.rb

```diff
@@ -1,6 +1,6 @@
 module CompilerConstants
-  GNU_GCC_VERSIONS = 3..9
-  GNU_GCC_REGEXP = /^gcc-(4\.[3-9])$/
+  GNU_GCC_VERSIONS = [4.3, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9, 5, 5.1]
```

Comment:
These should be strings, we don't want to use floating-point numbers for versions.